modules/delegates.js

```diff
@@ -220,26 +220,12 @@ __private.forge = function (cb) {
 		}
 
 		library.sequence.add(function (cb) {
-			async.series({
-				getPeers: function (seriesCb) {
-					return modules.transport.getPeers({limit: constants.maxPeers}, seriesCb);
-				},
-				checkBroadhash: function (seriesCb) {
-					var consensus = modules.peers.getConsensus();
-					if (modules.transport.poorConsensus(consensus)) {
-						return setImmediate(seriesCb, ['Inadequate broadhash consensus', consensus, '%'].join(' '));
-					} else {
-						return setImmediate(seriesCb);
-					}
-				}
-			}, function (err) {
-				if (err) {
-					library.logger.warn(err);
-					return setImmediate(cb, err);
-				} else {
-					return modules.blocks.process.generateBlock(currentBlockData.keypair, currentBlockData.time, cb);
-				}
-			});
+			var consensus = modules.peers.getConsensus();
+			if (modules.transport.poorConsensus(consensus)) {
```

Comment:
`modules.transport.poorConsensus` will call `modules.peers.getConsensus()` when no `consensus` supplied, so no need to call it before, or maybe we always should do that explicitly? If yes, then remove that call from `modules.transport.poorConsensus`.

Reply:
Consensus is assigned here to a separate variable so it can be reused in the log messages later.